### ISSUE TITLE
Demo code doesn't work with 3.0.0: TypeError: server.anyRequest is not a function

### ISSUE THREAD

**Issue by OnkelTem** [httptoolkit/mockttp-proxy-demo#1]
This demo uses 1.2.2 version of the library.
I tried upgrading it to the latest `mockttp` (3.0.0) and it doesn't work.

```
> mockttp-proxy-demo@1.0.0 start
> node index.js

/projects/custom/mitm/mockttp-proxy-demo/index.js:23
    .anyRequest()
     ^

TypeError: server.anyRequest is not a function
    at /projects/custom/mitm/mockttp-proxy-demo/index.js:23:6
```

The guide is not up-to-date as well: https://httptoolkit.tech/blog/javascript-mitm-proxy-mockttp/ as it's using `anyRequest()` too.

I didn't find any Changelog or Release notes describing this change.
I didn't find also any issues on this matter.

**Comment by pimterry** [comment]
That was quick! I just released v3 on Friday night, and I haven't updated everything yet. 

There are now detailed release notes available at https://github.com/httptoolkit/mockttp/releases/tag/v3.0.0 and I've updated the code in this repo: https://github.com/httptoolkit/mockttp-proxy-demo/commit/047756a153f2d3037c755579d74d0e6bb4997704. I'll update the article in a second too, do let me know if you run into any other issues. Thanks for the reminder!
